### PR TITLE
Enhancement/119

### PR DESCRIPTION
**Pull request by theflexbird**
#119 #120

### CODE REVIEW

File: src/client/actions/campaigns/getStatuses.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { CAMPAIGNS_STATS_LOAD_SUCCESS } from '../../constants/campaignStats';
+import request from 'superagent-bluebird-promise';
+import Promise from 'bluebird';
+
+export function getStatuses() {
+    return (dispatch, getState) => {
+        return request.get(
+            `${getState().currentService.location}/campaigns/1` //`${tenant}`
```
Comment (theflexbird) [inline src/client/actions/campaigns/getStatuses.js]: I know `1` is a fixed value. I didn't spot any information about id of company (tenant). There is no model or table corresponding to it.

Comment (celber) [inline src/client/actions/campaigns/getStatuses.js]: Isn't the customerId or companyId you are looking for?

Comment (theflexbird) [inline src/client/actions/campaigns/getStatuses.js]: From what I know company equal tenant. I already got info how to get it. Yes, I will change it.

File: src/server/db/models/Campaign.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ module.exports = function(sequelize) {
     },
     /** ISO 3166-1 company code. */
     companyId: {
-      type: Sequelize.STRING(30),
+      type: Sequelize.BIGINT(20),
```
Comment (celber) [inline src/server/db/models/Campaign.js]: it cant be BIGINT if its ISO 3166-1

Comment (theflexbird) [inline src/server/db/models/Campaign.js]: It will be repaired with #131. It is no point to fix it here.

Comment (celber) [inline src/server/db/models/Campaign.js]: Oh right, than change comment above please. It is confusing :)

**Pull request merged**